### PR TITLE
doc: convert to Markdown/MyST

### PR DESCRIPTION
**Pull request by ru-fu**
To be consistent with LXD documentation, move our docs to Markdown instead of RST.

### CODE REVIEW

**Comment by ru-fu** [comment]
The link check is expected to fail (because the MyST cheat sheet isn't on main yet).

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ This is just the beginning of MicroCloud. We’re very excited about what’s co
 
 ### **RESOURCES:**
 
-- Introduction: https://discuss.linuxcontainers.org/t/introducing-microcloud/15871
+- Documentation: https://canonical-microcloud.readthedocs-hosted.com/en/latest/
```
Comment (roosterfish) [inline README.md:61]: Not a blocker just wondering why we don't serve this under e.g. `documentation.ubuntu.com/microcloud`?

**Review by roosterfish — APPROVED** [review]
Thanks, looking great.

**Comment by roosterfish** [comment]
Merging it blocked (at least for me) until all the required checks pass.
Can we maybe merge something beforehand to let the test succeed?

File: README.md (hunk 1 above)
Comment (ru-fu) [reply README.md:61]: This would be great, but needs approval from @evildmp.

**Comment by ru-fu** [comment]
> Merging it blocked (at least for me) until all the required checks pass. Can we maybe merge something beforehand to let the test succeed?

Sure, I can open a PR to add only the MyST cheat sheet.

**Comment by roosterfish** [comment]
> Sure, I can open a PR to add only the MyST cheat sheet.

Can be rebase now, I have just merged #337.

**Comment by ru-fu** [comment]
Another attempt ...: https://github.com/canonical/microcloud/pull/339

**Comment by roosterfish** [comment]
> Another attempt ...: #339

Merged!

**Comment by roosterfish** [comment]
Mhh, the link checker tries to access `documentation.ubuntu.com/microcloud`. Shouldn't this be `canonical-microcloud.readthedocs-hosted.com`? I see the file is available here https://canonical-microcloud.readthedocs-hosted.com/en/latest/doc-cheat-sheet-myst/

**Review by roosterfish — APPROVED** [review]
LGTM!

**Pull request merged**